tools/acle_template.tex

```diff
@@ -57,104 +59,115 @@
   \renewcommand{\headrulewidth}{0pt}
 }
 
-$if(linestretch)$
-  \usepackage{setspace}
-  \setstretch{$linestretch$}
-$endif$
-
-$if(chapter-formatting)$
-  \usepackage{titlesec}
-  \titleformat{\chapter}[block]
-    {\normalfont\huge\bfseries}{\thechapter.}{1em}{\Huge}
-  \titlespacing*{\chapter}{0pt}{-19pt}{0pt}
-$endif$
-
-$if(indent)$
-$else$
-  \IfFileExists{parskip.sty}{%
-    \usepackage{parskip}
-  }{% else
-    \setlength{\parindent}{0pt}
-    \setlength{\parskip}{6pt plus 2pt minus 1pt}
-  }
-$endif$
-
 $if(highlighting-macros)$
   $highlighting-macros$
 $endif$
 
-$if(numbersections)$
-  \setcounter{secnumdepth}{$if(secnumdepth)$$secnumdepth$$else$5$endif$}
-$else$
-  \setcounter{secnumdepth}{0}
-$endif$
+\usepackage{titlesec}
+\titleformat{\chapter}[block]
+  {\normalfont\huge\bfseries}{\thechapter.}{1em}{\Huge}
+\titlespacing*{\chapter}{0pt}{-19pt}{0pt}
 
-$if(pagestyle)$
-  \pagestyle{$pagestyle$}
-$endif$
+\usepackage{parskip}
+\setlength{\parindent}{0pt}
+\setlength{\parskip}{6pt plus 2pt minus 1pt}
```

Comment:
Please add a comment along the lines of `set the indent level of the paragraphs`